src/Microsoft.ML.TensorFlow/doc.xml

```diff
@@ -0,0 +1,41 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+
+    <member name="TensorflowTransform">
+      <summary>
+        Extracts hidden layers' values from a pre-trained Tensorflow model.
+      </summary>
+      <remarks>
+        The TensorflowTransform extracts the specified output from the operation computed on the graph (given the input(s)) using a pre-trained <a href="https://www.tensorflow.org">Tensorflow</a> model.
+        The transform takes as input the Tensorflow model together with the names of the inputs to the model and name of the operation for which output values will be extracted from the model.
+
+        The TensorflowTransform has following assumptions regarding the input, output and processing of data.
+        <list type="number">
+          <item>
+            The transform currently accepts the <a href="https://www.tensorflow.org/mobile/prepare_models">frozen TensorFlow model</a> file as input.
+          </item>
+          <item>The transform supports scoring only one example at a time.</item>
+          <item>The name of input column(s) should match the name of input(s) in Tensorflow model.</item>
+          <item>The name of the output column should match one of the operations in the Tensorflow graph.</item>
+          <item>
+            Upon success, the transform will introduce a new column in <see cref="IDataView"/> based on the name of the output column specified.
+          </item>
+        </list>
+      </remarks>
```

Comment:
Should we add an explanation of the type of input that is expected? Maybe it is just to clarify that it has to be whatever format the TF model expects, or more detail regarding how images would need to be loaded through a different set of transforms.

Reply:
Do you mean a detailed sample?

---
In reply to: [213782041](https://github.com/dotnet/machinelearning/pull/704#discussion_r213782041) [](ancestors = 213782041)

Reply:
No, but just an explanation of what should be provided. If I'm new to image classification, how do I find out that I need to transform the images to be in the same format that the pretrained TF model expects? Maybe this is not the right place for this though.